### ISSUE TITLE
Rm production services intersection with SE

### ISSUE THREAD

**Issue by duker33** [fidals/stroyprombeton#238]
This prod services use the same ports as SE does:
```
stb-rabbitmq
stb-redis
stb-postgres (maybe)
stb-python-stage
```

Rm this ports intersection and check if services intersection problem gone.

**Services intersection problem**
```
$ cd stb/docker
$ dcp up -d
$ cd shopelectro/docker
$ dcp up -d
Recreating stb-rabbitmq
Recreating stb-redis
Recreating stb-postgres
Recreating stb-python-stage
...
```

**Comment by duker33** [comment]
good solution found here:
https://stackoverflow.com/questions/46582219/docker-compose-start-service-from-same-docker-compose-file-using-env-vars-to-al

**Comment by 0pdd** [comment]
@duker33 the puzzle [#404](https://github.com/fidals/stroyprombeton/issues/404) is still not solved; solved: [#255](https://github.com/fidals/stroyprombeton/issues/255).
